ccmbioinfo/lfs
# CARDS-1539: Trusted permissions mode: Users outside of the Trusted users group should not be seeing any errors in the Dashboard, Forms and Variants

To test:

- start with `-P cards4proms --permissions trusted`
- create two users, add one of them to the Trusted Users group
- log in with admin, make sure all the sidebar links work correctly
- log in with the untrusted user, make sure there is nothing in the sidebar, and the "pending approval" message is displayed
- log in with the trusted user, make sure all the sidebar links work correctly
